all/pom.xml

```
@@ -46,6 +46,7 @@
                 <artifactId>filevault-package-maven-plugin</artifactId>
                 <configuration>
                     <packageType>container</packageType>
+                    <name>acs-aem-commons-all</name>
```

```suggestion
                    <name>acs-aem-commons-all-classic</name>
```
I would make it more explicit that this is not suitable for cloud.

---

I would suspect that this would be a breaking change, on-premise customers who would use this new version would have two acs-aem-commons in their CRX Package Manager list, both on status installed. Not sure if this is wanted?

---

Right, let's leave it at the default.